src/main/steps/c100-rebuild/child-details/other-children/has-other-children/content.ts

```diff
@@ -0,0 +1,66 @@
+import { YesOrNo } from '../../../../../app/case/definition';
```

Comment:
Move has-other-children folder outside other-children and put it directly under child-details

Reply:
Sure makes sense for url